packages/cyberstorm/src/components/componentStyles/PackageFlag.module.css

```diff
@@ -0,0 +1,34 @@
+/* Common PackageFlag styles */
+.packageFlag {
+  display: flex;
+  flex: none;
+  flex-direction: row;
+  gap: var(--gap-xxxs);
+  align-items: center;
+  justify-content: center;
+
+  padding: var(--padding-xxxxs) var(--padding-xxxs);
+  border: var(--button-border-width) solid var(--button-border-color);
+  border-radius: 0 var(--size--xxxxxs) var(--size--xxxxxs) 0; /* Should we use sizes/em also here? */
+  color: var(--text-color);
+  background-color: var(--bg-color);
+
+  --text-color: var(--color-cyber-green);
+  --bg-color: var(--color-surface--3);
+  --button-border-color: transparent;
+  --button-border-width: var(--size--xxxxxxs);
+}
+
+/* Styles for PackageFlag's text */
+.packageFlagLabel {
+  padding-left: var(--size--xxxs);
+  font-weight: 800;
+  font-size: var(--size-xs); /* 12px instead of 11px because em roundings */
```

Comment:
The comment mostly confuses me? It should be either improved or removed if it's outdated.

Reply:
We did go this through on group and these comments were more of notes to go through than actual comments that should have gone on PR. I will delete these so no more confusion should be building up!

Other hand would be cool if there is tool or something to make personal notes instead of comments.